pyicloud/services/photos.py

```diff
@@ -0,0 +1,286 @@
+import sys
+import json
+import urllib
+
+from datetime import datetime
+from base64 import b64decode
+from bitstring import ConstBitStream
+from pyicloud.exceptions import PyiCloudBinaryFeedParseError
+
+
+class PhotosService(object):
+    """ The 'Photos' iCloud service."""
+
+    def __init__(self, service_root, session, params):
+        self.session = session
+        self.params = dict(params)
+
+        self.prepostfetch = 200
+
+        self._service_root = service_root
+        self._service_endpoint = '%s/ph' % self._service_root
+
+        request = self.session.get(
+            '%s/startup' % self._service_endpoint,
+            params=self.params
+        )
+        response = request.json()
+        self.params.update({
+            'syncToken': response['syncToken'],
```

Comment:
I've just tried running this a moment ago and I get a `KeyError` because `syncToken` wasn't in the response; the response, after checking, was apparently:

```
{u'errorCode': 402, u'requestUUID': u'4d12d29e-10c8-823a-2cbb-862d9502b72a', u'errorReason': u'Payment Required'}
```

The request itself was a `402`; perhaps you need to have paid for iCloud (or some other service) to get access to that feature.  Perhaps we could raise a helpful exception here if we didn't get a reasonable response code?